src/editor/editorconfig.jsdoc

```diff
@@ -144,7 +144,9 @@
  *			toolbar: {
  *				items: [ 'bold', 'italic', '|', 'undo', 'redo' ],
  *
- *				viewportTopOffset: 30
+ *				viewportTopOffset: 30,
+ *
+ *				shouldGroupWhenFull: true
```

Comment:
```suggestion
 *				shouldGroupWhenFull: false
```

No one will be setting it to `true` as this is the default.

Reply:
Actually, I'd rather rename it to `shouldNotGroupWhenFull` to avoid a flag which is default by true. Then you'll also have less issues with handling the default in the other two PRs.